app/models/account.rb

```diff
@@ -58,6 +58,27 @@ def self.columns
   include States
   include ProviderDomains
 
+  self.background_deletion = {
+    users: { action: :destroy },
```

Comment:
I dunno if we should make that the default action is 'destroy' and we don't need to repeat `action: :destroy` everywhere 😄

Reply:
In that case, it would be:
```ruby
  self.background_deletion = [
    :users,
    :mail_dispatch_rules,
    # many other here
    buyer_accounts: { class_name: 'Account' },
    # many other here
    profile: { action: :delete },
    # many other here
  ].freeze
```

Reply:
let me think about it..